Dialog/SpritesEditorDialog.cpp

```diff
@@ -341,6 +360,107 @@ LevelComponents::EntitySet *SpritesEditorDialog::GetCurEntitySetPtr(bool createN
     return curEntityset;
 }
 
+/// <summary>
+/// Render OAM Set based on the ListView data
+/// </summary>
+/// <param name="selectrow">
+/// select a row after re-render oam graphic, leave it blank to set default value 0
+/// </param>
+void SpritesEditorDialog::RenderOamSet(int selectrow)
+{
+    int rowcount = ListViewItemModel->rowCount();
+    if (!rowcount) return;
+    unsigned short *nakedOAMData = new unsigned short[rowcount * 3];
```

Comment:
Do not use new for this

Reply:
no, its size will change every time

Reply:
That is fine. You should only use dynamic memory if the data should persist after this function. If it is used only in this function, then do not use `new`

Reply:
So, please change this to `unsigned short nakedOAMData[rowcount * 3];`

Reply:
done with QVector and for loop